dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassSymbolReference.java

```diff
@@ -32,6 +32,18 @@ public String getErrorString() {
     return getTargetClassName() + " is not found, referenced from " + getSourceClassName();
   }
 
+  /**
+   * Creates an instance from {@code source} and {@code symbol}.
+   */
+  static ClassSymbolReference fromSymbol(ClassFile source, ClassSymbol symbol) {
+    // This method is for the refactoring (#574).
+    return builder()
+        .setTargetClassName(symbol.getClassName())
+        .setSourceClassName(source.getClassName())
+        .setSubclass(symbol instanceof SuperClassSymbol)
```

Comment:
again, this reads very weird

Reply:
`subClass` is a flag to indicate the reference is class inheritance. "subClass" will be gone once we finish the refactoring.

Reply:
OK, I think I get it now. Still not clear in the code though.

Reply:
Added comment. (I thought updating field name `isSubclass` but the class is to be removed soon after this refactoring)